docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,572 @@
+
+- **Kubernetes Cluster**: A functioning Kubernetes cluster with kubectl access.
+- **Helm**: Helm CLI installed
+- **DNS Configuration**: Access to configure DNS for your domain to point to the Kubernetes cluster ingress.
+- **TLS Certificates**: Obtain valid X.509 certificates for your domain from a trusted Certificate Authority.
+- **External Dependencies**: Provision the following external dependencies:
+  - **Amazon Aurora PostgreSQL**: For persistent data storage required for the Web Modeler component.
+  - **Amazon OpenSearch**: For indexing and analytics.
+  - **AWS Simple Active Directory**: For authentication and authorization.
+- **NGINX Ingress Controller**: Ensure the NGINX ingress controller is set up in the cluster.
+- **AWS OpenSearch Snapshot Repository** - This will be a place to store the backups of the Camunda cluster. This repository must be configured with OpenSearch to take backups.
+- **s3 Bucket** - This will be used to store backups of the Camunda cluster. This s3 bucket must be configured with OpenSearch to take backups.
+- **Persistent Volumes**: Configure block storage persistent volumes for stateful components.
+- **Resource Planning**: Evaluate sufficient CPU, memory, and storage necessary for the deployment. Have a look at our [sizing guide](/components/best-practices/architecture/sizing-your-environment.md/#camunda-8-self-managed) for more information.
+
+Ensure all prerequisites are in place to avoid issues during installation or upgrading in a production environment.
+
+## Architecture Overview
+
+Below is the high-level architecture diagram for the base production setup _(click on the image to open the PDF version)_:
+[![Infrastructure Diagram ROSA Single-Region](./assets/smarch.jpg)](./assets/smarch.pdf)
+
+- Supported Components:
+  Camunda Platform Orchestration Core Components: Zeebe, Tasklist, Operate, Optimize
+- Ingress Controller (e.g., Nginx)
+- External Dependencies:
+  - PostgreSQL (for persistent data storage)
+  - AWS OpenSearch (for indexing and analytics)
+  - Keycloak (for authentication/authorization)
+
+# Step-by-Step Production Guide
+
+## Installation and Configuration
+
+#### Namespace Setup
+
+To get started, create two namespaces:
+
+```bash
+kubectl create namespace management
+kubectl create namespace orchestration
+```
+
+Within the `management` namespace, we will install Identity, Console, and all the Web Modeler components. Within the `orchestration` namespace, we will install the Camunda Orchestration Core component, along with Connectors and Optimize importer. We do not have to worry about installing each component separately since that will be handled by the Helm Chart automatically.
+
+#### Installing the Helm Chart
+
+The Camunda Helm Chart can be installed using the following command:
+
+```bash
+# This will install the latest Camunda Helm chart with the latest applications/dependencies.
+helm install camunda camunda/camunda-platform \
+    --values production-values.yaml
```

Comment:
I'd recommend mentioning probably both of the following

```
helm repo add camunda https://helm.camunda.io
```

```
helm repo update
```

depends a bit.
If it's a new user, they may not have added the helm chart repo yet.
If it's an existing user, they may not have updated it in a while.

Reply:
+1

Reply:
added the commands, thanks